frontstage/views/surveys/upload_survey.py

```diff
@@ -34,37 +34,50 @@ def upload_survey(session):
     party_controller.is_respondent_enrolled(party_id, business_party_id, survey_short_name)
 
     # Get the uploaded file
-    upload_file = request.files['file']
-    upload_filename = upload_file.filename
+
+    # The variable name has '_raw' so that it can be called in the file size checking function,
+    # and not interfere with the actual upload function
+
+    upload_file_raw = request.files['file']
+    upload_filename = upload_file_raw.filename
+
     upload_file = {
-        'file': (upload_filename, upload_file.stream, upload_file.mimetype, {'Expires': 0})
+        'file': (upload_filename, upload_file_raw.stream, upload_file_raw.mimetype, {'Expires': 0})
     }
 
-    try:
-        # Upload the file to the collection instrument service
-        collection_instrument_controller.upload_collection_instrument(upload_file, case_id, party_id)
-    except CiUploadError as ex:
-        if ".xlsx format" in ex.error_message:
-            error_info = "type"
-        elif "50 characters" in ex.error_message:
-            error_info = "charLimit"
-        elif "File too large" in ex.error_message:
-            error_info = 'size'
-        else:
-            logger.error('Unexpected error message returned from collection instrument service',
-                         status=ex.status_code,
-                         error_message=ex.error_message,
-                         party_id=party_id,
-                         case_id=case_id)
-            error_info = "unexpected"
+    # Checking if the file size is great than 5999 bytes
```

Comment:
The comment and error message check that it's 6kb+ but the function just checks for it being more than 0.
Also, how did we arrive to 6kb?  It sounds about right and I assume you just made an empty excel file and looked at the size of it?